aliases/available/general.aliases.bash

```diff
@@ -18,6 +18,11 @@ alias l1='ls -1'
 
 alias _="sudo"
 
+# Shortcuts to edit startup files
+alias vigen="vim ~/.bash_it/aliases/available/general.aliases.bash"
```

Comment:
Lately, yes :) However I'm certainly open to suggestions on how to more quickly edit files.